src/index.js

```diff
@@ -5,15 +5,18 @@ import App from "./App";
 import { makeServer } from "./server";
 import { BrowserRouter } from "react-router-dom";
 import { NoteProvider } from "./Context/notes-context";
+import { AuthProvider } from "./Context/auth-context";
 // Call make Server
 makeServer();
 
 ReactDOM.render(
   <React.StrictMode>
     <BrowserRouter>
-      <NoteProvider>
-        <App />
-      </NoteProvider>
+      <AuthProvider>
+        <NoteProvider>
+          <App />
+        </NoteProvider>
+      </AuthProvider>
```

Comment:
And instead of adding the providers in index.js you can create a new file called as CombinedProviders.jsx

```
import React from "react";
import { AuthProvider, NoteProvider } from "LOCATION WHERE CONTEXTS ARE STORED";

const CombinedProvider = ({ children }) => {
    return (
         <AuthProvider>
        <NoteProvider>
          {children}
        </NoteProvider>
      </AuthProvider>
    )
}

export { CombinedProvider };
```